frontend/public/components/monitoring.tsx

```diff
@@ -517,26 +532,52 @@ const alertStateToProps = (state: RootState, { match }): AlertsDetailsPageProps
   const { loaded: silencesLoaded }: Silences = silencesToProps(state);
   const ruleID = _.get(match, 'params.ruleID');
   const labels = getURLSearchParams();
-  const alerts = _.filter(data, (a) => a.rule.id === ruleID);
+  const alerts = _.filter(data, (a) => a?.rule?.id === ruleID);
   const rule = _.get(alerts, '[0].rule');
   const alert = _.find(alerts, (a) => _.isEqual(a.labels, labels));
-  return { alert, loaded, loadError, rule, silencesLoaded };
+  const activePerspective = getActivePerspective(state);
+  const namespace = getActiveNamespace(state);
+  const url = match.url;
+  return { alert, loaded, loadError, rule, silencesLoaded, activePerspective, namespace, url };
 };
 
-const AlertsDetailsPage = withFallback(
+export const AlertsDetailsPage = withFallback(
   connect(alertStateToProps)((props: AlertsDetailsPageProps) => {
-    const { alert, loaded, loadError, rule, silencesLoaded } = props;
+    const {
+      alert,
+      loaded,
+      loadError,
+      rule,
+      silencesLoaded,
+      activePerspective,
+      namespace,
+      url,
+    } = props;
     const { annotations = {}, labels = {}, silencedBy = [] } = alert || {};
     const { alertname, severity } = labels as any;
     const state = alertState(alert);
-
     return (
       <>
         <Helmet>
           <title>{`${alertname} · Details`}</title>
         </Helmet>
         <StatusBox data={alert} label={AlertResource.label} loaded={loaded} loadError={loadError}>
-          <div className="co-m-nav-title co-m-nav-title--detail">
+          <div
+            className={classNames('co-m-nav-title co-m-nav-title--detail', {
+              'co-m-nav-title--breadcrumbs': activePerspective === 'dev',
+            })}
+          >
+            {activePerspective === 'dev' && (
+              <BreadCrumbs
```

Comment:
This ```BreadCrumbs``` is going to be for both the perspective.

Reply:
👍 Yes, we want to add breadcrumbs to the admin view too.